### DISCUSSION TITLE
How can I make authentik redirect straight to a source, rather than displaying the login screen?

### DISCUSSION THREAD

**Discussion by aicreed** [goauthentik/authentik#5437] [Q&A] [answered]
Hi everyone,

I am using Authentik as a wrapper for Danish MitID (https://mitid.dk) and I am using MitID as the sole authenticator, username and password login is disabled, so the only login option shown to users is MitID source. However, I would like Authentik to just automatically choose this source for the user, rather than displaying a login screen with the source on it.

This is what users currently see, I want it to just directly redirect to the source:
![image](https://user-images.githubusercontent.com/47678215/235434024-9ba0636d-1da3-47c9-b841-1e4614aaa1b4.png)

Thanks,
Adam.

**Comment by remram44** [comment, marked as answer]
If you configure your identification stage to have no user field and only one source, it will redirect to it automatically. See PR #5583

Source: https://www.tekonline.com.au/authentik-quick-tip-direct-social-authentication-without-login-form/
